src/main/java/com/harrison/cb/CouchbaseClient.java

```diff
@@ -0,0 +1,79 @@
+package com.harrison.cb;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.Cluster;
+import com.couchbase.client.java.CouchbaseCluster;
+import com.couchbase.client.java.document.JsonDocument;
+import com.couchbase.client.java.document.json.JsonArray;
+import com.couchbase.client.java.document.json.JsonObject;
+import com.couchbase.client.java.query.N1qlQuery;
+import com.couchbase.client.java.query.N1qlQueryResult;
+import com.couchbase.client.java.query.N1qlQueryRow;
+
+@Service
+public class CouchbaseClient {
+	
+	
+	public List<String> doCbStuff() {
+		Cluster cluster = CouchbaseCluster.create("localhost");
+		Bucket bucket = cluster.openBucket("default");
+		
+	    // Create a JSON Document
+	    JsonObject arthur = JsonObject.create()
+	        .put("name", "Arthur")
+	        .put("email", "kingarthur@couchbase.com")
+	        .put("interests", JsonArray.from("Holy Grail", "African Swallows"));
+	    
+	    System.out.println(bucket.get("u:king_arthur"));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "u:king_arthur" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-54-187-222-128.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=squid%3AS1192)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-54-187-222-128.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=squid%3AS106)